bin/getConfig.js

```diff
@@ -11,8 +12,8 @@ function getConfig() {
     return firefoxConfig;
   }
 
-  const envConfig = process.env.TARGET === "firefox-panel" ?
-                  firefoxConfig : developmentConfig;
+  const envConfig = process.env.TARGET === "application" ?
+                  applicationConfig : developmentConfig;
```

Comment:
You just got rid of `firefoxConfig`. You need to handle the `firefox-panel` target here.